teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(_id, name, specialization, age = 'unknown')
+    @specialization = specialization
+    super(age, name)
+  end
```

Comment:
- [ ] Kindly take note of the details.

  - `_id` is not being used any longer, kindly remove it.
  - name should have 'unknown', not age

  ```
  def initialize(specialization, age, name = 'unknown')
    @specialization = specialization
    super(age, name)
  end
  ```